golem_messages/message/p2p.py

```diff
@@ -141,3 +141,13 @@ class SetTaskSession(base.Message):
         'conn_id',
         'super_node_info',
     ] + base.Message.__slots__
+
+
+class RemoveTaskContainer(base.Message):
+    TYPE = P2P_MESSAGE_BASE + 17
+
+    __slots__ = ['remove_task'] + base.Message.__slots__
```

Comment:
Could we make it `remove_tasks` list? Even if for now we will put only single message there.